app/assets/javascripts/modules/mobile-filters-modal.js

```diff
@@ -2,104 +2,57 @@ window.GOVUK = window.GOVUK || {}
 window.GOVUK.Modules = window.GOVUK.Modules || {};
 
 (function (Modules) {
-  function MobileFiltersModal ($module) {
-    this.$module = $module
-    this.$facetsBox = this.$module.querySelector('.facets__box')
-    this.$closeTriggers = this.$module.querySelectorAll('.js-close-filters')
-    this.$showResultsButton = this.$module.querySelector('.js-show-results')
-    this.$clearFiltersTrigger = this.$module.querySelector('.js-clear-selected-filters')
-    this.$body = document.querySelector('body')
-
-    this.$module.open = this.handleOpen.bind(this)
-    this.$module.close = this.handleClose.bind(this)
-    this.$module.clearFilters = this.handleClearFilters.bind(this)
-    this.$module.ModalFocus = this.handleModalFocus.bind(this)
-    this.$module.boundKeyDown = this.handleKeyDown.bind(this)
+  function MobileFiltersModal (module) {
+    this.module = module
+    this.triggerElement = document.querySelector('[data-toggle="mobile-filters-modal"][data-target="' + this.module.id + '"]')
+    this.clearFiltersTrigger = this.module.querySelector('.js-clear-selected-filters')
+    this.module.toggle = this.handleToggle.bind(this)
+    this.module.open = this.handleOpen.bind(this)
+    this.module.close = this.handleClose.bind(this)
```

Comment:
Not sure where to put this comment, so adding here to allow you to reply. I wanted to ask about the behaviour of the `Show search results` button. It seems to be a jump link to the search results immediately below it, which makes sense now I've thought about it but initially confused me as I assumed it would hide the search filters. Is this intended behaviour?

Reply:
@andysellick Yep this is intentional - at first I removed the button entirely, and even though the results update automatically the designer thought it's better to have a button which jumps to the results.

Reply:
Thanks. I think now that this might be a significant enough change that we should do some user testing to make sure we've got it right. That would make this piece of work a lot bigger and more complex though. I'll put a note on the trello card and we can discuss it further offline.

Reply:
@andysellick  The UI changes were posted in the accessibility community, and there were no major issues found - initially there were some changes requested, but they were rescinded after further investigation. Therefore, it has had some internal testing already. Before it was rescinded, one of the suggestions was to change the text to something like "Go to search results" - maybe we changing the text would be useful? Or "Jump to search results" or "Skip to search results"?

Reply:
`Go to search results` sounds good, let's change it to that.